### PR TITLE
Add custom document of lsp-latex.

### PR DESCRIPTION
**Pull request by ROCKTAKEY**
@ericdallo 
Thanks for your maintainance. 
I have made `lsp-latex.md`. Could you review this? Is it right?

Related to #1814.

Thanks!

### CODE REVIEW

**Comment by ericdallo** [comment]
Nice, I'll test it local shortly

File: docs/page/lsp-latex.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+cargo install --git https://github.com/latex-lsp/texlab.git
+```
+### Debugger: Not available
+
+## Installation(client)
+You can get `lsp-latex` from melpa. Add melpa to the list `package-archives`,
+and run:
+```
+M-x package-initialize M-x package-install lsp-latex
+```
```
Comment (ericdallo) [inline docs/page/lsp-latex.md:18]:
```suggestion
<kbd>M-x</kbd> `package-initialize` <kbd>M-x</kbd> `package-install` <kbd>RET</kbd> `lsp-latex`
```

File: docs/page/lsp-latex.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+## Available configurations
+These are available configuration. All configurations texlab provides is available.
+See also [document of texlab](https://texlab.netlify.app/docs/reference/configuration).
```
Comment (ericdallo) [inline docs/page/lsp-latex.md:22]:
```suggestion
These are the available configurations. All configurations `texlab` provides are available.
See also [document of texlab](https://texlab.netlify.app/docs/reference/configuration).
```

File: docs/page/lsp-latex.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+## Installation(client)
+You can get `lsp-latex` from melpa. Add melpa to the list `package-archives`,
+and run:
+```
+M-x package-initialize M-x package-install lsp-latex
+```
+
+## Available configurations
+These are available configuration. All configurations texlab provides is available.
```
Comment (ericdallo) [inline docs/page/lsp-latex.md:21]:
```suggestion
These are the available configurations. All configurations `texlab` provides are available.
```

**Comment by ROCKTAKEY** [comment]
Thanks! The suggestions are all right. I will merge these suggestions soon.

**Comment by ROCKTAKEY** [comment]
@ericdallo All suggestions are commited. Now it can be merged.

**Comment by ericdallo** [comment]
Thank you for your contribution @ROCKTAKEY!

**Comment by ROCKTAKEY** [comment]
Thank you, too, @ericdallo!

**Pull request merged**